src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -13,5 +13,6 @@
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     // to set restrictions as some operations can only be applied to some entity
     public static final String MESSAGE_INVALID_TYPE = "The type of entity is invalid";
+    public static final String MESSAGE_INVALID_INDEX = "Index is not a non-zero unsigned integer.";
```

Comment:
The double negative seems a little confusing, how about MESSAGE_INVALID_INDEX = "Invalid index! Index should be a non-zero unsigned integer." haha just throwing out ideas :)